hugegraph-dist/src/main/java/com/baidu/hugegraph/cmd/InitStore.java

```diff
@@ -83,7 +87,15 @@ public static void main(String[] args)
                         "Must contain at least one graph");
 
         for (ConfigurationNode graphName : graphNames) {
+            String name = ((Map.Entry<String, Object>)
+                           graphName.getReference()).getKey();
```

Comment:
can we call graphName.getName()

Reply:
graphName.getName() returns 'graph' due to it's sub node of 'graphs'